ext/json/tests/json_decode_exceptions.phpt

```diff
@@ -0,0 +1,50 @@
+--TEST--
+Test json_decode() function : JSON_THROW_ON_ERROR flag
+--FILE--
+<?php
+
+try {
+    var_dump(json_decode("{", false, 512, JSON_THROW_ON_ERROR));
```

Comment:
tab is used as indent for all json tests (at least those that I'm aware of) so it would be nice to keep ti consistent.